src/models/financialNonpayment.js

```diff
@@ -0,0 +1,55 @@
+const nonpaymentInfractionOptions = [
+  'Repossession',
+  'Defaulted',
+  'Collections',
+  'Cancelled',
+  'Evicted',
+  'Garnished',
+  'Delinquent',
+  'Any',
+]
+
+const financialNonpayment = {
+  Name: { presence: true, hasValue: true },
+  Infractions: {
+    presence: true,
+    array: {
+      validator: {
+        presence: true,
+        inclusion: nonpaymentInfractionOptions,
+      },
+      length: { minimum: 1 },
+    },
+  },
+  AccountNumber: { presence: true, hasValue: true },
+  PropertyType: { presence: true, hasValue: true },
+  Amount: {
+    presence: true,
+    hasValue: {
+      validator: {
+        numericality: {
+          greaterThan: 0,
+        },
+      },
+    },
+  },
+  Reason: { presence: true, hasValue: true },
+  Status: { presence: true, hasValue: true },
+  Date: {
+    presence: true,
+    date: { requireDay: false },
+  },
+  Resolved: (value, attributes) => {
```

Comment:
Date will not validate if a date is added to the fields. It will only validate if "Not resolved" is chosen.